client/packages/programs/src/VaccineCourseEditModal/VaccineCourseEditModal.tsx

```diff
@@ -114,18 +114,25 @@ export const VaccineCourseEditModal: FC<VaccineCourseEditModalProps> = ({
   };
 
   const save = async () => {
-    const agesAreInOrder = (draft.vaccineCourseDoses ?? []).every(
-      (dose, index, doses) => {
-        const prevDoseAge = doses[index - 1]?.minAgeMonths ?? -0.01;
-        return dose.minAgeMonths > prevDoseAge;
-      }
-    );
+    const doses = draft.vaccineCourseDoses ?? [];
+
+    const agesAreInOrder = doses.every((dose, index, doses) => {
```

Comment:
hmm kinda double references to `doses` whoops. I assume the arrow function scope uses the param one rather than the upper scope one added...

Reply:
This validation is otherwise just formatting changes.